sample/Makefile.macos

```diff
@@ -16,6 +16,3 @@ all:
 	clang++ $(ALLFLAGS) derivative.cc -lmujoco.2.1.3        -o derivative
 	clang++ $(ALLFLAGS) basic.cc      -lmujoco.2.1.3 -lglfw -o basic
 	clang++ $(ALLFLAGS) record.cc     -lmujoco.2.1.3 -lglfw -o record
-	clang -c $(CFLAGS) uitools.c
```

Comment:
I do not have a way to test the macos makefiles. I could do it for windows if I setup a new development environment, but that would take a noticeable amount of time.

Assistance with the windows and macos makefiles would be very welcome!